eng/common/TestResources/New-TestResources.ps1

```diff
@@ -515,7 +515,7 @@ is based on the cloud to which the template is being deployed:
 
 .PARAMETER Environment
 Name of the cloud environment. The default is the Azure Public Cloud
-('PublicCloud')
+('AzureCloud')
```

Comment:
FYI this touches eng/common so will need to be updated with tools process.

Reply:
I only proposed "AzureCloud" since that's how it's referred to in other places. If that's too risky given the downstream changes required, please disregard.

Reply:
I think it's just the parameter documentation that's out of date, not any functionality (actual values default to `AzureCloud`). Fix: https://github.com/Azure/azure-sdk-tools/pull/1127